docs/designers-developers/developers/slotfills/plugin-sidebar.md

```diff
@@ -0,0 +1,34 @@
+# PluginSidebar
+
+This slot allows for adding items into the Gutenberg Toolbar. This is most commonly used for plugins and a great example is the Yoast SEO plugin.
```

Comment:
```suggestion
This slot allows for adding items into the Gutenberg Toolbar.
```

I didn't find any reference to other plugins in the docs, so I'd rather be conservative and don't add this here.